modules/installation-creating-aws-worker.adoc

```diff
@@ -48,8 +54,8 @@ template requires:
     "ParameterValue": "ami-<random_string>" <4>
   },
   {
-    "ParameterKey": "WorkerSubnet", <5>
-    "ParameterValue": "yes" <6>
+    "ParameterKey": "PrivateSubnet", <5>
+    "ParameterValue": "sg-<random_string>" <6>
```

Comment:
Copy/paste error? Obviously here should be "Subnet" as "ParameterKey" per CF template change.

Reply:
Thank you, @jianlinliu! The dev source changed again after I edited this file. I am fixing this error.

Reply:
A minor suggestion, seen from the "sg-<random_string>", looks like "ParameterValue" is asking some security group name for "Subnet", if possible, pls make the example value for "ParameterValue" more meaningful, but not confuse user.

Reply:
You're right. I changed it to "subnet-<random-string>." Do you see any other changes?

Reply:
lgtm now.